build.zig.zon

```diff
@@ -1,46 +1,27 @@
 .{
     .name = "microzig",
-    .version = "0.13.0",
-    .minimum_zig_version = "0.13.0",
+    .version = "0.0.0",
     .dependencies = .{
-        // packages within the monorepo so that others can reach them
-        .build = .{ .path = "build" },
-        .@"build/definitions" = .{ .path = "build/definitions" },
-        .core = .{ .path = "core" },
-        .@"tools/regz" = .{ .path = "tools/regz" },
-        .@"tools/uf2" = .{ .path = "tools/uf2" },
-        .@"port/nordic/nrf5x" = .{ .path = "port/nordic/nrf5x" },
-        .@"port/nxp/lpc" = .{ .path = "port/nxp/lpc" },
-        .@"port/microchip/atsam" = .{ .path = "port/microchip/atsam" },
-        .@"port/microchip/avr" = .{ .path = "port/microchip/avr" },
-        .@"port/gigadevice/gd32" = .{ .path = "port/gigadevice/gd32" },
-        .@"port/stmicro/stm32" = .{ .path = "port/stmicro/stm32" },
-        .@"port/espressif/esp" = .{ .path = "port/espressif/esp" },
-        .@"port/raspberrypi/rp2xxx" = .{ .path = "port/raspberrypi/rp2xxx" },
+        .@"microzig/build-internals" = .{ .path = "build-internals" },
 
-        // examples so that we can build them all in one go
```

Comment:
If you're going to put all the example packages under one, please add the dependency here.

Reply:
I tried this and it gives me a strange error. Otherwise, building all the examples with `zig build` works fine.
```
/home/toto/dev/zig-embedded/microzig/examples/build.zig:1:1: error: file exists in multiple modules
/home/toto/dev/zig-embedded/microzig/examples/build.zig:1:1: note: root of module root.@dependencies.122004ac89e8f0168f24759576ced3d80b47ccd0c0fe4d923e83794d8a813b026932
/home/toto/dev/zig-embedded/microzig/examples/build.zig:1:1: note: root of module root.@build
/home/toto/dev/zig-embedded/microzig/examples/../examples/raspberrypi/rp2xxx/build.zig:1:1: error: file exists in multiple modules
/home/toto/dev/zig-embedded/microzig/examples/../examples/raspberrypi/rp2xxx/build.zig:1:1: note: root of module root.@dependencies.122036a1b1203ff16a15cdf6e496f567225abc589ef4295bf109cca8758b3c27e304
/home/toto/dev/zig-embedded/microzig/examples/../examples/raspberrypi/rp2xxx/build.zig:1:1: note: root of module root.@dependencies.122016b96a0fddd1a5ea10a2f969decf63be550014831b01cc906b441c300f15c801
/home/toto/dev/zig-embedded/microzig/examples/nxp/lpc/build.zig:1:1: error: file exists in multiple modules
/home/toto/dev/zig-embedded/microzig/examples/nxp/lpc/build.zig:1:1: note: root of module root.@dependencies.12208201167b38fd4b7c7b676e2a973d198ec63f809045b8a55654f72d5dbf4c9e21
/home/toto/dev/zig-embedded/microzig/examples/nxp/lpc/build.zig:1:1: note: root of module root.@dependencies.122020eda6f6de553d128be12ae72e608c7f060f091ee0920f2348da5a0fafbc6dc3
/home/toto/dev/zig-embedded/microzig/examples/../examples/stmicro/stm32/build.zig:1:1: error: file exists in multiple modules
/home/toto/dev/zig-embedded/microzig/examples/../examples/stmicro/stm32/build.zig:1:1: note: root of module root.@dependencies.12207ea8a92f85718d537bfc361d17e2010c2c612598825adf035e2bd23e57df342d
/home/toto/dev/zig-embedded/microzig/examples/../examples/stmicro/stm32/build.zig:1:1: note: root of module root.@dependencies.1220461576a7eef1ad0195ef9b32c632712429bb6e6cc6f32a6757eaf66995c720ab
```

Reply:
That's fair then, I'm not sure what exactly the issue is but we can leave it for now